systest/backup/filesystem/backup_test.go

```diff
@@ -113,7 +112,6 @@ func TestBackupOfOldRestore(t *testing.T) {
 
 	// Clean the cluster and try restoring the backups created above.
 	testutil.DropAll(t, dg)
-	time.Sleep(2 * time.Second)
```

Comment:
Don't we need these anymore? The DropAll could take a few minutes to be effective.

Reply:
I think this should not be needed. Why would DropAll take few minutes to be effective?

Reply:
I will remove these sleep for now. If they are needed, or tests become flaky, then we can bring them back.